readthedocs/proxito/urls.py

```diff
@@ -108,6 +108,19 @@
         name='docs_detail',
     ),
 
+    # Hack /en/latest so it redirects properly
+    # We don't want to serve the docs here,
+    # because it's at a different level of serving so relative links break.
+    url(
+        (
+            r'^(?:projects/(?P<subproject_slug>{project_slug})/)?'
+            r'(?P<lang_slug>{lang_slug})/'
+            r'(?P<version_slug>{version_slug})$'.format(**pattern_opts)
+        ),
+        ServeDocs.as_view(redirect_root=True),
```

Comment:
From the tests, I understand that the "root" is not the problem, but "directory indexing" is. I would suggest to adapt the name of the variable and comment related to match the logic.

Reply:
Also, if we are short-cutting the flow immediately in the `.get` method, there is no need to use `ServeDocs` here. We can just use `handler_404`/`fast_404` and return 404 directly.

Reply:
True 👍